src/styles/components/multiple-form/styles.less

```diff
@@ -42,17 +42,30 @@
   }
 
   .permission-form {
-    min-height: 325px;
-  }
 
-  .review-config,
-  .multiple-form {
-    height: 462px;
+    .modal-body-wrapper {
```

Comment:
This class is not in the PermissionBuilderModal, because it is being overridden, and `permission-form` is being applied to same element as `modal-body-wrapper` would be.

Even after this, it seems that modal needs a little ❤️
![](https://cl.ly/2G1z2u343B0E/Image%202016-10-05%20at%2015.20.02.png)